src/smt/command.cpp

```diff
@@ -1353,12 +1353,10 @@ std::string GetQuantifierEliminationCommand::getCommandName() const throw() {
 GetUnsatCoreCommand::GetUnsatCoreCommand() throw() {
 }
 
-GetUnsatCoreCommand::GetUnsatCoreCommand(const std::map<Expr, std::string>& names) throw() : d_names(names) {
-}
-
 void GetUnsatCoreCommand::invoke(SmtEngine* smtEngine) {
   try {
     d_result = smtEngine->getUnsatCore();
+    // copy names here instead of storing the pointer?
```

Comment:
Not sure I understand this comment.

Reply:
This is left over from before, I'll remove